SQL-2103: Publish signing keys for JDBC (#268)
mongodb/mongo-jdbc-driver: README.md

You can download the precompiled driver (jars) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).  
Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is `com.mongodb.jdbc.MongoDriver`.

### Verify Integrity of MongoDB JDBC Driver Packages

> To verify the integrity of the JDBC Driver packages, the artifact signature needs to be verified. To do this, there are two steps:

Reviewer: ```suggestion
To verify the integrity of the JDBC Driver packages, follow the steps below:
```

1. Import the public key:  

> - You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.  

Reviewer 2: are we sure all 3 of these are correct? mit and ubunutu worked for me but openpgp gives "gpg: keyserver receive failed: No data"

Reviewer: This is so odd, Ethan had the same issue. It worked fine for me.

> - To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 30666110`.  

Reviewer 2: nit: we can probably merge this and the following example into a single bullet

> - You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.  
> - To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 30666110`.  
> - For example, `gpg --keyserver keys.openpgp.org --recv-keys 30666110`. 

Reviewer: ```suggestion
The public key ID is 91A2157730666110.   

To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 91A2157730666110`. 
Replace <server url> with one of the current GPG Keyservers supported by Maven Central Servers:
    - keyserver.ubuntu.com
    - keys.openpgp.org
    - pgp.mit.edu
```

2. Verify artifact:  

> - For this step, you will need to create a local copy of the `.jar.asc` file from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc) in addition to downloading the `.jar` file.  
> - To verify the signature, run the following command `gpg --verify mongodb-jdbc-<version>.jar.asc mongodb-jdbc-<version>.jar`  
> - For example, `gpg --verify mongodb-jdbc-2.1.2.jar.asc mongodb-jdbc-2.1.2.jar`.

Reviewer: ```suggestion
- Download the artifact you want to verify along with the matching detached signature `.asc` file, from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).
 For example, mongodb-jdbc-2.1.2.jar and mongodb-jdbc-2.1.2.jar.asc.
- To verify the signature, run the following command `gpg --verify <detached_signature_file> <artifact_to_verify>`.
 For example, `gpg --verify mongodb-jdbc-2.1.2.jar.asc mongodb-jdbc-2.1.2.jar`.
```
